internal/pkg/workspace/workspace.go

```diff
@@ -60,13 +60,13 @@ const (
 // Summary is a description of what's associated with this workspace.
 type Summary struct {
 	Application string `yaml:"application"` // Name of the application.
-	Path        string `yaml:"-"`           // absolute path to the summary file.
+	Path        string `yaml:"-"`           // Absolute path to the summary file.
 }
 
 // Workspace typically represents a Git repository where the user has its infrastructure-as-code files as well as source files.
 type Workspace struct {
 	workingDirAbs string
-	copilotDirAbs string
+	CopilotDirAbs string
```

Comment:
we can make this private now! 😊

Reply:
I couldn't get WX's suggestion to work, so leaving it for a future refactor.